src/Features/LanguageServer/ProtocolUnitTests/CodeActions/CodeActionResolveTests.cs

```diff
@@ -23,8 +24,8 @@ public CodeActionResolveTests(ITestOutputHelper testOutputHelper) : base(testOut
         {
         }
 
-        [WpfFact]
-        public async Task TestCodeActionResolveHandlerAsync()
+        [WpfTheory, CombinatorialData]
```

Comment:
>  none of the test of these tests can even notice.

But some of hte tests did notice, and we only found that out by updating all of them.   This approach at least means if someone makes a new test and it fires for one or hte other that we realize there's a difference.

I do recognize this is a lot of duplication.  But it really *does* ensure that the test operates the same way for a non-mutating host (like VS) as well as a mutating host (in case we ever get one of those).